### PR TITLE
latest

### PR DESCRIPTION
**Pull request by ojokure**
##  What does this PR do
as a user I want to be able to reset my password via email

##  Description of task to be completed

password reset/account recovery

##  How should this manually be tested

click on forgot password and follow the prompt 

##  What are the relevant Trello board stories

as a user I want to be able to change my password 

##  Questions (If any):

### CODE REVIEW

File: src/resources/users/user-controllers.js [hunk 1]
```diff
@@ -25,13 +25,15 @@ exports.accountRecovery = async (req, res) => {
...
+      error: {
+        message: 'password reset link is invalid or has expired',
+      },
     });
   }
 };
 
 exports.resetPasswordEmail = async (req, res) => {
-  const { email } = req.body;
+  const email = req.body.email;
```
Comment (cyrus-za) [inline src/resources/users/user-controllers.js:36]: why?

Comment (ojokure) [reply src/resources/users/user-controllers.js:36]: No particular reason actually ...more readability for someone without destructuring knowledge I guess

Comment (cyrus-za) [reply src/resources/users/user-controllers.js:36]: If there's someone on the team that does not understand the basics of ES2015, I'd rahter skill that person up than to refactor code

Comment (ojokure) [reply src/resources/users/user-controllers.js:36]: Touche

**Pull request merged**